src/activity-functions-function-notation.xml

```diff
@@ -58,19 +53,25 @@
         <p>Start by remebmering this: one input, one output.</p>
 
         <p>A <term>function</term><index><main>function</main></index> is a rule that takes a piece of <q>usable</q> information that you pick (input) and gives you some kind of result (output).  By <q>usable</q> we mean information that has meaning for the function.</p>
-        <p>A simple example is that of the outside temperature throughout the day in your neighborhood. You pick a time to walk outside, like <m>10</m> a.m. That is an input. The result of making that choice may be an outside temperature of <m>42</m> degress fahrenheit. That is an output.</p>
-
-        <p>But no matter what time you pick to go outside, there is only one temperature. It's not like it can be <m>42</m> degrees and <m>75</m> degrees at the same time.  Also, using 24 hour time, you can choose to go outside at <m>10</m> hours or <m>15</m> hours. But you can't go outside at <m>37</m> hours because it makes no sense.  There are only <m>24</m> hours in a day.  The input of <m>37</m> hours is not usable.</p>
-
-        <p>Before we go further in defining a function, we need a short to write and talk about functions.  We need some notation so we don't have to keep writing all those letters, F U N C T I O N, that's just way too many letters.</p>
+        <p>Think of a function as a sort of machine.  You put something into the machine, the machine does something then out pops a result.  In this course we will mostly put numbers into the machine and the machine will give us other numbers as results.</p>
+        <p>A simple example of a function machine is a temperature predictor.  The machine will predict the outside temperature throughout the day in your neighborhood. You only have to choose a time to walk outside, like <m>10</m> a.m. That choice of time is an input. The result of making that choice can is an outside temperature like <m>42</m> degress fahrenheit. That result is an output. The function in this case is the rule or process the machine uses to predict the temperature given a time of day.</p>
+        <p>In this example we say the temperature outside is a <term>function</term> of the time of day. It's a function because there is a relationship that links time and temperature together and most importantly, for each time you choose there an be only one temperature predicted.</p>
+        <p>This brings up some good questions:</p>
+        <ul>
```

Comment:
A `<ul>` (and `<ol>`, `<dl>`) should generally be inside a `<p>`. This was not always the case, but has been settled for maybe six months or more, and is how things will stay.  Exceptions are things that are always going to have one of these, like `<objectives>` and `<list>`. In those situations, you need no `<p>`.